### ISSUE TITLE
O> organize translations in folders

### ISSUE THREAD

**Issue by michaelizer** [rchain/bounties#351]
I suggest to create new folders for the translations instead of naming them FILENAME_LANGUAGE.md, like rchain/Members/Languages/ES/ or rchain/Members/Translations/ES/

**Comment by Viraculous** [comment]
@michaelizer,  I edited this issue for better organizing.

**Comment by BelovedAquila** [comment]
@michaelizer that's a nice suggestion, do you have steps and plans on how to make that a reality than just suggestions?

**Comment by michaelizer** [comment]
@BelovedAquila done, I created (check these pull requests [342](https://github.com/rchain/Members/pull/342) and [343](https://github.com/rchain/Members/pull/343)) Members/Language/ES for a better organization, you could create new internal folders for Tagalog (Language/TG) and move Tagalog files there.

**Comment by BelovedAquila** [comment]
@michaelizer alright.
 @David405 consider this issue please,can you collaborate on it?

**Comment by dckc** [comment]
I found a [Feb 2017 article][1] that recommends:

```
mysite/
├ _i18n/
  ├ de/
    └ about.md
  ├ en/
    └ about.md
  └ ...
├ about.md
└ ...
```

[1]: http://www.klaasnotfound.com/2017/02/16/proper-multilingual-site-with-github-pages-and-jekyll/

**Comment by BelovedAquila** [comment]
@michaelizer and this issue clicks to that of @Viraculous on issue #243 ,since you both had same suggestions sort of ,you can also collaborate to work on it.

**Comment by BelovedAquila** [comment]
#243

**Comment by David405** [comment]
Hello @michaelizer and @BelovedAquila, I like this development, and has seen your pull requests. I believe all translations in a particular language should be kept in the same folder for ease of access, what do you guys think?

**Comment by BelovedAquila** [comment]
@David405 same thoughts or suggestions, but we are not on thinking or suggesting, we are on finding measures to accomplish it, can that be achieved?

**Comment by David405** [comment]
We should create child folders for translated documents for each language here on github rchain parent folder. What I mean simply is that there will be a rchain/spanish folder, rchain/italian folder, rchain/greek folder and so on, this will enable people who only have basic proficiency or even entirely non-english speakers to easily access documents in their language. 

Also, we should encourage people translation documents to save them in the appropriate folders.

**Comment by BelovedAquila** [comment]
@david405 you are still making suggestions, let the work begin.

**Comment by David405** [comment]
I know that, I wanted to know everyone's else take on it before starting. Currently on mobile, will create some new folders when on my laptop

**Comment by ICA3DaR5** [comment]
@dckc That's a really nice tree. See: https://github.com/decred/dcrdocs and https://github.com/decred/dcrdocs/tree/master/src/i18n

**Comment by David405** [comment]
I believe we need to get the persons with admin priviledges or have access to master to get involved in this issue.

**Comment by dckc** [comment]
I don't see anything that needs admin privileges. But I'm not inclined to approve nor merge any translation PRs until someone steps up to do better quality control of translations. See #302.

@ICA3DaR5 it' reasonably clear that you're skilled in Greek; perhaps you'd like to give some feedback on #377? Note that it doesn't follow the pattern of using 2 letter ISO language abbreviations as used in decred and the `_i18n` tree above.

**Comment by David405** [comment]
@dckc at least there should be some documents that have already been reviewed and accepted as quality work. We can start from there. What we want to achieve in this issue is organization of documents in a particular into a single folder

**Comment by David405** [comment]
@dckc at least there should be some documents that have already been reviewed and accepted as quality work. We can start from there. What we want to achieve in this issue is organization of documents in a particular into a single folder

**Comment by David405** [comment]
at least there should be some translated documents that are satisfactory, we could start by organizing those ones into a common folder depending on their language.

**Comment by ICA3DaR5** [comment]
@David405 Let's use the standard _i18n tree as it's used on the Decred project.

See: https://github.com/decred/dcrdocs and https://github.com/decred/dcrdocs/tree/master/src/i18n

Also, we could have some of the documents sync with some translation service (like Transifex), this is going to make localization much easier, and the Members won't have to use GitHub for translations.

We might get some fake translations on Transifex though since the barrier to entry it's easier.

I have translated most of the documentation and the previous site of the Decred project to Greek. This is just my experience.

My 2¢.

**Comment by David405** [comment]
Nice @ICA3DaR5, do we have a project existing already on transifex or do we have to create one? if there is, kindly send the link, so others can join in and also I am all in for the i18n tree standard

**Comment by ICA3DaR5** [comment]
@David405 As far as I know, we don't.

Let's get the OK from @dckc and @lapin7 an then do the i18n tree standard then?

**Comment by Keaycee** [comment]
@David405 You've got a point. Creating a parent folder of different languages about RChain would make it much easy for members, who do not understand English or other languages to be able read and know more about RChain. @michaelizer Thank you for coming up with this thread.

**Comment by David405** [comment]
@lapin7 please respond to @ICA3DaR5 most recent comment.

**Comment by lapin7** [comment]
I can only think 2 seconds about:
_i18n or Transifex
So that's nearly nothing. I don't have even a clue about what you're talking.

@ICA3DaR5 and @David405 seem to know a lot more about it.
So go ahead and try get more collaborators behind your ideas.
It would be great if you reach consensus about the best way of working. And get other collaborators to instruct new-comers about this way of working.

**Comment by dckc** [comment]
I expect that most translations don't belong in this repository. They belong in the repository of the translated document. So I'm not sure there's much to generalize here.

In any case, I don't see much to evaluate. I suggest closing this issue as `wontfix` (or is it `invalid` here?) and working on individual documents.

**Comment by dckc** [comment]
@ICA3DaR5 I just merged #397 ...

But now I'm having second thoughts: I wonder if `README_i18n` is a better folder name than `README`. Maybe not... `i18n` is somewhat obscure.

I'll close this and see if everybody agrees. (anybody who does not agree: feel free to re-open it.)
